tests_pr/homework.py

```diff
@@ -0,0 +1,124 @@
+import string
+from math import log
+
+
+def list_compare(list1, list2):
+    return list(set(list1) & set(list2))
+
+
+def count_letter_in_text(text, letter):
```

Comment:
please, documenting your code

Reply:
what do you mean? insert comment with tasks?